### PR TITLE
added new solana-tps contract

### PR DESCRIPTION
**Pull request by thegreat-vanderlinde**

### CODE REVIEW

File: src/pages/contract/solana-tps/index.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+			const depositEnd = moment().add(2, 'days').toDate().getTime();
+			const settleStart = moment('2022-12-31 09:00:00Z').toDate().getTime();
+
+			const initParams: OtcInitializationParams = {
+				reserveMint: new PublicKey('7XSvJnS19TodrQJSbjUR6tEGwmYyL1i9FX7Z5ZQHc53W'),
+				depositStart,
+				depositEnd,
+				settleStart,
+				seniorDepositAmount: 1000,
+				juniorDepositAmount: 1000,
```
Comment (thegreat-vanderlinde) [inline src/pages/contract/solana-tps/index.tsx:62]: in the description we have 100 but here we're using 1000 @FedererKK

Comment (FedererKK) [reply src/pages/contract/solana-tps/index.tsx:62]: Yes, I've increase the default Deposit amount for this contract as TPS is in the thousands range, also:
- in the prompt I ask the user to 'Choose a tx number from from 1000 to 4000 to select'
- default airdrop is 1000devUSD
so it should not affect negatively anything

Comment (thegreat-vanderlinde) [reply src/pages/contract/solana-tps/index.tsx:62]: yes, my concern is that [here](https://github.com/FedererKK/vyper-otc-ui/blob/640b46133168fede970cdf5309d1ae4cd1fde7b6/src/pages/contract/solana-tps/index.tsx#L117) in the description we're declaring a deposit of 100 and in the end [here](https://github.com/FedererKK/vyper-otc-ui/blob/640b46133168fede970cdf5309d1ae4cd1fde7b6/src/pages/contract/solana-tps/index.tsx#L61-L62) during contract creation we're using 1000. maybe we should set 1000 also in the description?

Comment (FedererKK) [reply src/pages/contract/solana-tps/index.tsx:62]: Yikes, good catch.. Yes, it should be 1000 there too

**Comment by vantessel** [comment]
closing as we can open in separate PR the new contract

**Pull request closed without merging**